ERCS/erc-4337-rpc.md

```diff
@@ -0,0 +1,542 @@
+---
+eip:
```

Comment:
```suggestion
eip: 7769
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.